src/utils.jl

```diff
@@ -174,7 +47,22 @@ function fullmatrix(M::MPO; reverse::Bool = true)
   return array(permute(Mmat, c', c))
 end
 
-fullmatrix(L::LPDO; kwargs...) = fullmatrix(MPO(L); kwargs...)
+function ITensors.array(L::LPDO{MPO}; reverse::Bool = true) 
```

Comment:
Same thing here, I think when the MPO is Choi-like, it should be a separate function `choi_to_array`.

Reply:
I would rather have only one function that can be applied to all objects we deal with, rather than having different names

Reply:
Another alternative to having different function names `array` and `choi_to_array` would be defining an unexported PastaQ function called `array`, which then gets called as `PastaQ.array`. Then we can have custom behavior which is hard coded to work with commonly used PastaQ objects, like Choi-like MPOs and LPDOs (along with standard MPOs and LPDOs).